en_us/quests/puppetfile.md

```diff
@@ -0,0 +1,235 @@
+{% include '/version.md' %}
+
+# Puppetfile
+
+## Quest objectives
+
+- Create a Puppetfile to manage external module dependencies
+- Use the `puppet module tool` to calculate dependencies and help determine
```

Comment:
italicize instead of mark as code, perhaps (since puppet module tool is not a command)?